static-checks/nist-validation/test.py

```diff
@@ -27,12 +30,15 @@
         '-valresultfile', result_file,
         '-file', datastream,
     ]
-    proc = util.subprocess_run(cmd, stdout=subprocess.PIPE, check=True, universal_newlines=True)
-    if 'The target is valid' in proc.stdout:
+    util.subprocess_run(cmd, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, check=True)
```

Comment:
Why mute stderr? What if the tool fails for some unknown reason and we won't be able to debug it?

Does it produce too much output on stderr?

Reply:
normally it doesn't produce anything on stderr

but it produces a lot on stdout

Reply:
Good point, I will update it to still print stderr.